.evergreen/config.yml

```diff
@@ -631,3 +686,10 @@ buildvariants:
   run_on: rhel70
   tasks:
     - .versioned-api
+
+- matrix_name: "serverless"
+  matrix_spec: { "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
+  display_name: "Serverless"
+  run_on: rhel70
```

Comment:
Didn't you mention possibly cutting off RHEL testing? Any reason not to use Debian or Ubuntu here?

Reply:
None except that we have most on rhel70. I'm currently collecting the operating systems and versions actually used and adapting the build hosts to reflect reality. For the time being, I chose rhel70 based on the config I shamelessly copied and adapted.